### PR TITLE
feat(preview-middleware): add LocalStorageConnector

### PR DESCRIPTION
**Pull request by zdravko-georgiev**
Adds LocalStorageConnector to `sandbox.html` so that app personalizations are kept across preview sessions
#1158

### CODE REVIEW

File: .changeset/lucky-rocks-juggle.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/preview-middleware': patch
+---
+
+Adds LocalStorageConnector to sandbox.html
```
Comment (tobiasqueck) [inline .changeset/lucky-rocks-juggle.md:5]: could we please change it "Adds local persistence of personalizations across local preview sessions" or something along the lines to better explain what the change does?

**Review by tobiasqueck — APPROVED** [review]
* suggestion for improving the changeset added, however, it is not critical, just a suggestion
* changes adhere to coding guidelines and seem to be the best option out of (https://sapui5.hana.ondemand.com/sdk/#/topic/642dab291a7b47ec9d46c39b3c482aba.html)
* did NOT locally test it

File: .changeset/lucky-rocks-juggle.md (hunk 1 above)
Comment (zdravko-georgiev) [reply .changeset/lucky-rocks-juggle.md]: Thanks, done with https://github.com/SAP/open-ux-tools/pull/1381/commits/53c6787393b5f67aee391b4bb895885ab3d15787

**Review by tobiasqueck — APPROVED** [review]

**Review by johannes-kolbe — APPROVED** [review]
- Changes look good and concise.
- Adds the parameter as required in BLI
- Changeset provided
- Did not test manually

**Pull request merged**